### ISSUE TITLE
ECA study program - Partner requirements

### ISSUE THREAD

**Issue by nielsraes** [GBIF-Europe/eca-study-program-discussionboard#2]
Would it be possible to provide some information under which conditions partners can join and what the benefits would be. The Dutch GBIF node is embedded in Naturalis Biodiversity Center which is not a Higher Education Institution (HEI). The start document states that "The consortium should involve at least 3 institutions from at least 3 different countries, and multiple associated partners from the academic (e.g. these can be GBIF nodes with academic affiliations) and non-academic world (e.g. **GBIF nodes that are not HEI**)". How will the non-academic partners be compensated for their contributions to the study program?

**Comment by JuliaPawlowska** [comment]
Yes. I'm working on this;) Just need a little bit of time but will provide it before summer break.

**Comment by JuliaPawlowska** [comment]
I would like to let you know that 4EU+ EMDM proposal was not funded but we received good feedback and at least some of us would like to reapply next February with more specific application called for the moment "quantitative biodiversity". 

The nice thing is that for the next call we can apply as a completely new consortium. So I would like to know who is interested in potentially joining consortium? 

In the EMDM (Erasmus Mundus Design Measure: https://erasmus-plus.ec.europa.eu/opportunities/opportunities-for-organisations/cooperation-among-organisations-and-institutions/erasmus-mundus-design-measures-emdm?facets__field_eac_tags=185) application consortium members must be HEI (higher education institution) and an associate partner can be any institution. EMDM are awarded in form of lamp sum 60k euros for 15 months for leading institution, which is in charge of spending them for whole consortium to achieve defined goals. EMJM (Erasmus Mundus Joint Master https://erasmus-plus.ec.europa.eu/opportunities/opportunities-for-organisations/cooperation-among-organisations-and-institutions/erasmus-mundus-joint-masters?facets__field_eac_tags=186) is bigger deal. Project budget can go up to €5 million per project for 6 years. Here each consortium member (must be HEI) will have separate budget. Organizations not being HEIs can't have their own budget, but can be invited for teaching and be payed from budget of full consortium members.

As far as I know, when a specific node is affiliated/hosted at the univeristy/acedemia/technical school (in HEI) this hosting institution should officially sign a consortium agreement. It is completely internal to decide between node and HEI how this can be arranged. If a specific node is not affiliated with HEI then it can be only associate partner:(

I think it would be good to think about specific ideas and how to proceed further. We could start to work on common proposal in early October, reusing some experience we already have from previous call.

**Comment by JuliaPawlowska** [comment]
There was a certain twist and I'm happy to let you know that the 4EU+ Erasmus Mundus Design Measure "BioMS: Joint Master in Biodiversity" proposal was finally funded. The project agreement between the University of Warsaw and EC was signed last week and the project officially starts on December 1st. The ambition is to establish an international joint master study program bringing together competencies in studying biodiversity in space and over time with modern digital skills needed for big data integration, analysis, and modeling. Internally, our goal is to prepare the Erasmus Mundus Joint Master application for February 2026 in consortium including all HEIs and associate partners interested in joining the endeavor. We plan to organize a first meeting in Warsaw, in January 2025. If you are interested in joining the discussion, let me know.
